src/assets/CollectibleDetectionController.ts

```diff
@@ -228,6 +229,7 @@ export class CollectibleDetectionController extends BaseController<
     this.defaultConfig = {
       interval: DEFAULT_INTERVAL,
       networkType: MAINNET,
+      chainId: '1',
```

Comment:
Nit: I don't object to doing this for now, since the default `networkType` is `MAINNET` already, so this is a pre-existing issue. But we shouldn't be setting a default network at all, we can get the true current network from the network controller by passing in a `getNetworkState` function to the constructor.

This could lead to bugs where something runs initially before the first network state update, and uses the wrong network.